src/app/forms/components/checkbox-group/checkbox-group.component.spec.ts

```diff
@@ -11,14 +11,15 @@ import { CheckboxGroupComponent } from './checkbox-group.component';
 @Component({
```

Comment:
There should be tests where you have the value already loaded i.e. edit mode so no need to click but rather set the value and assert that corresponding checkboxes are selected

Reply:
this is what the test on line 62 does...`it('should be propagated from the form control to the element', () => {`